UPGRADE-4.0.md

```diff
@@ -1,6 +1,16 @@
 UPGRADE FROM 3.x to 4.0
 =======================
 
+## Some classes and methods were marked as final, thus they cannot be overridden:
```

Comment:
This list only contains classes, what are the deprecated methods?

Reply:
Fixed.